cos-fleetshard-sync/src/main/java/org/bf2/cos/fleetshard/sync/resources/ConnectorNamespaceProvisioner.java

```diff
@@ -13,10 +14,13 @@
 import org.slf4j.LoggerFactory;
 
 import io.fabric8.kubernetes.api.model.Namespace;
+import io.fabric8.kubernetes.api.model.Secret;
 import io.fabric8.kubernetes.client.utils.KubernetesResourceUtil;
 
 @ApplicationScoped
 public class ConnectorNamespaceProvisioner {
+    public static final String ADDON_PULLSECRET_NAME = "addon-pullsecret";
```

Comment:
this should be made configurable (`addon-pullsecret` should be the default)

Reply:
I made it configurable now with a similar config like in the Debezium Operator using `application.properties`.

Reply:
can we add a small test to verify that it work also with a different config name  ?

Reply:
Done